src/components/localization/localized-link.js

```diff
@@ -141,6 +141,8 @@ const affiliate_links = ['affiliate_sign_in', 'affiliate_sign_up']
 const deriv_app_links = ['dbot', 'deriv_app', 'mt5', 'derivx']
 const deriv_other_products = ['binary', 'smart_trader']
 const deriv_social_platforms = ['blog', 'community', 'developers', 'zoho']
+// add item to this array if you need to make an internal link open on a new tab without modal window
+const new_tab = ['terms-and-conditions']
```

Comment:
```suggestion
const new_tab_no_modal = ['terms-and-conditions']
```
Currently, adding items to this only prevents new modal from opening. You should add the functionality of opening in a new tab by using checks using this constant instead of manually passing it via the `target` props for better abstraction

Reply:
ok